services/wallet/bridge/hop.go

```diff
@@ -40,14 +43,14 @@ type HopTxArgs struct {
 }
 
 type BonderFee struct {
-	AmountIn                *big.Int `json:"amountIn"`
-	Slippage                float32  `json:"slippage"`
-	AmountOutMin            *big.Int `json:"amountOutMin"`
-	DestinationAmountOutMin *big.Int `json:"destinationAmountOutMin"`
-	BonderFee               *big.Int `json:"bonderFee"`
-	EstimatedRecieved       *big.Int `json:"estimatedRecieved"`
-	Deadline                int64    `json:"deadline"`
-	DestinationDeadline     int64    `json:"destinationDeadline"`
+	AmountIn                *bigint.BigInt `json:"amountIn"`
```

Comment:
why not using "normal" big int

Reply:
Cause some values that we're receiving from the Hop may be `null` (like `destinationAmountOutMin`, `deadline`, `destinationDeadline`) and since `bigint.BigInt`handled that in its unmarshal function decided to use it for all values in this struct.